examples/cosmos/lightclient/Lightclient.qnt

```diff
@@ -0,0 +1,569 @@
+/**
+ * A state-machine specification of the lite client verification,
+ * following the English spec:
+ *
+ * https://github.com/informalsystems/tendermint-rs/blob/master/docs/spec/lightclient/verification.md
```

Comment:
For me too, but since I just copied it from the TLA file, I have no idea where the ENG spec is supposed to be now. Maybe @konnov knows the up-to-date URL.